ci/jenkins/test-vmc.sh

```diff
@@ -315,22 +333,53 @@ function deliver_antrea {
     if [[ "$COVERAGE" == true ]]; then
         docker save -o antrea-ubuntu-coverage.tar antrea/antrea-ubuntu-coverage:${DOCKER_IMG_VERSION}
     else
-        docker save -o antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu:${DOCKER_IMG_VERSION}
+        docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:${DOCKER_IMG_VERSION}
```

Comment:
I may be missing something, but I don't understand this change. `projects.registry.vmware.com/antrea/antrea-ubuntu` is the "main" image name we use now. We still tag the image as `antrea/antrea-ubuntu` but we may drop that in the future.

Reply:
Changed image name prefix to "projects.registry.vmware.com".